config/routes.rb

```diff
@@ -1,5 +1,20 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  get '/a' => 'application#root'
+  root :to => 'restaurant#index'
+
+  resources :restaurant do
```

Comment:
After you make the controller name changes I wrote above, this should change to `resources :restaurants`